### ISSUE TITLE
I used to have your filter on firefox, but had to re-download the browser and re-do the add ons. Now I can't modify my profanity list on firefox. What am I doing wrong?

### ISSUE THREAD

**Issue by 77trv** [richardfrost/AdvancedProfanityFilter#38]
Issues are used primarily to track bugs in the project. Feature requests can be submitted either as an issue or [here](https://goo.gl/forms/LTqFpJ0mCTsrgGgf2).

### Details (Bugs Only)

Chrome Version:
Extension Version:
Operating System:
Site(s) Affected:

### Steps to reproduce (Bugs only)

1.
2.
3.
4.

**Comment by 77trv** [comment]
I had to re-install firefox and re-add the profanity filter, and now it won't let me modify my profanity list. I click on it and it shows this: here's a screenshot. 
![image](https://user-images.githubusercontent.com/34660822/34124425-db06d804-e400-11e7-9376-2c80d8effda1.png)

**Comment by richardfrost** [comment]
Sorry for the slow response, been busy the last few months. I haven't seen that behavior before, and I just tested on a clean Firefox install and have no problems modifying the profanity list? If you are still having the issue, I would love to get to the bottom of it!

**Comment by 77trv** [comment]
Because of the filter, I switched to Chrome, so I haven't looked at it in a while. I opened Firefox today and now it works. It says it was updated on the 18th of December - maybe that fixed it, or maybe it was my computer. Either way, it's working now. It's a very good filter and I'm glad it's available.Thanks for your response.Virginia Talbot

    On ‎Wednesday‎, ‎10‎ ‎January‎, ‎2018‎ ‎02‎:‎44‎:‎20‎ ‎AM‎ ‎EST, Richard Frost <notifications@github.com> wrote:  
 
 
Sorry for the slow response, been busy the last few months. I haven't seen that behavior before, and I just tested on a clean Firefox install and have no problems modifying the profanity list? If you are still having the issue, I would love to get to the bottom of it!
